sdk/communication/azure-communication-administration/azure/communication/administration/_phone_number_administration_client.py

```diff
@@ -80,520 +45,139 @@ def from_connection_string(
-
-        :keyword list[str] phone_numbers: The list of phone numbers in the release request.
-        :keyword str continuation_token: A continuation token to restart a poller from a saved state.
-        :rtype: ~azure.core.polling.LROPoller[~azure.communication.administration.PhoneNumberRelease]
-        """
-        cont_token = kwargs.pop('continuation_token', None)  # type: Optional[str]
-
-        release_polling = ReleasePhoneNumberPolling(
-            is_terminated=lambda status: status in [
-                ReleaseStatus.Complete,
-                ReleaseStatus.Failed,
-                ReleaseStatus.Expired
-            ]
-        )
-
-        if cont_token is not None:
-            return LROPoller.from_continuation_token(
-                polling_method=release_polling,
-                continuation_token=cont_token,
-                client=self._phone_number_administration_client.phone_number_administration
-            )
-
-        if "phone_numbers" not in kwargs:
-            raise ValueError("Either kwarg 'phone_numbers' or 'continuation_token' needs to be specified")
-
-        create_release_response = self._phone_number_administration_client.\
-            phone_number_administration.release_phone_numbers(
-                **kwargs
-        )
-
-        initial_state = self._phone_number_administration_client.phone_number_administration.get_release_by_id(
-            release_id=create_release_response.release_id
-        )
-
-        return LROPoller(client=self._phone_number_administration_client.phone_number_administration,
-                         initial_response=initial_state,
-                         deserialization_callback=None,
-                         polling_method=release_polling)
-
-    @distributed_trace
-    def list_all_releases(
-            self,
-            **kwargs  # type: Any
-    ):
-        # type: (...) -> ItemPaged[PhoneNumberEntities]
-        """Gets a list of all releases.
-
-        :keyword int skip: An optional parameter for how many entries to skip, for pagination purposes.
-        The default is 0.
-        :keyword int take: An optional parameter for how many entries to return, for pagination purposes.
-        The default is 100.
-        :rtype: ~azure.core.paging.ItemPaged[~azure.communication.administration.PhoneNumberEntities]
-        """
-        return self._phone_number_administration_client.phone_number_administration.get_all_releases(
-            **kwargs
-        )
-
-    @distributed_trace
-    def get_reservation_by_id(
-        self,
-        reservation_id,  # type: str
-        **kwargs  # type: Any
-    ):
-        # type: (...) -> PhoneNumberReservation
-        """Get reservation by reservation id.
-
-        :param reservation_id: The reservation id to get reservation.
-        :type reservation_id: str
-        :rtype: ~azure.communication.administration.PhoneNumberReservation
+        # type: (...) -> ItemPaged[AcquiredPhoneNumbers]
```

Comment:
This should be `ItemPaged[AquiredPhoneNumber]`? I don't think we need the collective object?